scripts/mcptam_install.bash

```diff
@@ -1,8 +1,25 @@
 #!/bin/bash
+# Normal Use: ./mcptam_install.bash /path/to/catkin_ws 
+# CI Use: ./mcptam_install.bash /path/to/catkin_ws CI
+# Note: You must set up a catkin workspace in advance and either launch this script
+# from the workspace or pass the path to the workspace. Uses catkin_make to build 
+# mcptam.
+
 set -e  # exit on first error
-ROS_VERSION="indigo"
-ROS_BASH="/opt/ros/$ROS_VERSION/setup.bash"
 
+# Determine workspace location
+CATKIN_WS=$1
+CI=false
+if [ $# -eq 2 ] && [ $2 == "CI" ];	then
```

Comment:
OK, did you one better, set default action to assume current dir is catkin workspace, but still accept arg for changing it to another dir, so script can be launched from anywhere, and check to make sure dir selected is actually a workspace.